### ISSUE TITLE
train loss: nan

### ISSUE THREAD

**Issue by bwee** [jbohnslav/deepethogram#36]
Hi, 

Thanks so much for putting this out there for us to use, it's been very helpful.

I recently tried to set up a new project, where I imported several labeled videos from a previous project through the Video > Add Multiple button in the GUI. I then copy/pasted the .csv label files from the previous project into the individual folders for each video in the DATA directory. 

The feature extractor seems to be working but in the terminal it says "train loss: nan: x%" and counts up like normal, then says "val loss: nan: x%. The loss figure on page 1 of classification_metrics.tiff also says "nan" instead of plotting a value.

Do you have any advice?

Thanks again!

**Comment by bwee** [comment]
Figured it out -- I had no frames labeled as background.
